bedrock/exp/templates/exp/firefox/accounts-2019.html

```diff
@@ -0,0 +1,161 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. #}
+
+{% from "macros-protocol.html" import feature_card with context %}
+{% from "macros.html" import fxa_email_form, fxa_link_fragment, monitor_button with context %}
+
+{% extends "exp/base/base-firefox.html" %}
```

Comment:
Same as #7834 this page gets the Mozilla footer instead of the Firefox one, the base needs updating.

Reply:
I'll rebase this branch once #7834 merges 👍